### ISSUE TITLE
[Accessibility: Images of text] Plugins logos do not have a text representation

### ISSUE THREAD

**Issue by alinefm** [kimchi-project/wok#76]
On Firefox > View > Page Style > No Style

All images should be replaced by a text.
In the login page, the plugins logo keep being shown as images.
